docs/guides/data-replication.mdx

```diff
@@ -0,0 +1,261 @@
+
+
+import CreateReplicaCli from "!!raw-loader!./examples/cli/data_replication_create.sh"
+import CreateReplicaPy from "!!raw-loader!./examples/py/src/data_replication_create.py"
+import CreateReplicaJs from "!!raw-loader!./examples/js/src/data_replication_create.mjs"
+import CreateReplicaRs from "!!raw-loader!./examples/rs/examples/data_replication_create.rs"
+import CreateReplicaCpp from "!!raw-loader!./examples/cpp/src/data_replication_create.cc"
+import CreateReplicaCurl from "!!raw-loader!./examples/curl/data_replication_create.sh"
+import CreateReplicaProvision from "!!raw-loader!./examples/provisioning/data_replication_create.yml"
+
+<Tabs>
+    <TabItem value="CLI">
+        <CodeBlock language="bash">
+            {CreateReplicaCli}
+        </CodeBlock>
+    </TabItem>
+    <TabItem value="Web Console">
+        Steps to create a replication task using the Web Console:
+        1. Open the Web Console at **[http://127.0.0.1:8383](http://127.0.0.1:8383)** in your browser.
+        2. Enter the API token if the authorization is enabled.
+        3. Click on the **"Replication"** tab in the left sidebar.
+        4. Click on the plus icon in the top right corner to create a new replication task:
+        ![Create Replication Task](./img/data_replication_create.webp)
+        5. In the **"Create a new replication"** dialog, entre the name of the replication name and settings:
+         <img src={require("@site/docs/guides/img/data_replication_settings.webp").default} alt="Create Replication Task Dialog" width="300"/>
+        6. Click on the **"Create Replication"** button to create the replication task.
+    </TabItem>
+    <TabItem value="Python">
+        <CodeBlock language="python">
+            {CreateReplicaPy}
+        </CodeBlock>
+    </TabItem>
+    <TabItem value="JavaScript">
+        <CodeBlock language="javascript">
+            {CreateReplicaJs}
+        </CodeBlock>
+    </TabItem>
+    <TabItem value="Rust">
+        <CodeBlock language="rust">
+            {CreateReplicaRs}
+        </CodeBlock>
+    </TabItem>
+    <TabItem value="C++">
+        <CodeBlock language="cpp">
+            {CreateReplicaCpp}
+        </CodeBlock>
+    </TabItem>
+    <TabItem value="Curl">
+        <CodeBlock language="bash">
+            {CreateReplicaCurl}
+        </CodeBlock>
+    </TabItem>
+    <TabItem value="Provisioning">
+        <CodeBlock language="yaml">
+            {CreateReplicaProvision}
+        </CodeBlock>
+    </TabItem>
+</Tabs>
+
+### Browse Replication Tasks
+
+You can list all replication tasks and get detailed information about a specific replication task using the ReductStore SDKs, REST API, CLI and Web Console.
+The detailed information includes status, current settings and statistics of the replication task:
+
+* **Status**: The status of the replication task. It can be `Active` or `Inactive`. Inactive replication tasks are paused and don't replicate data usually because the target database is unreachable.
```

Comment:
the is_provisioned flag could be used here:
- Status:
- Provisioned:
- Number of Pending Records:
- ...